replica/database.cc

```diff
@@ -842,6 +845,9 @@ void database::init_schema_commitlog() {
     c.extensions = &_cfg.extensions();
     c.use_o_dsync = _cfg.commitlog_use_o_dsync();
     c.allow_going_over_size_limit = true; // for lower latency
+    if (features().fragmented_commitlog_entries) {
+        c.allow_fragmented_entries = true;
+    }
```

Comment:
Doesn't this mean we'll wait until the next restart?

Better to update it post-fact for use on all new segments.

Reply:
Do you mean on runtime config change, or do you mean later in startup?

Reply:
When the last node in the cluster is upgraded, other nodes should see allow_fragmented_entries = true.

Could be by making the configuration variable an updateable_value\<bool> and updating it.

Reply:
Need to both make commitlog config at least partially updatable in runtime + add feature listener for that really.

Reply:
We have that in a few places already.